Generation/src/components/MomentumRangeParticleGun.cpp

```diff
@@ -9,12 +9,7 @@
 #include "GaudiKernel/PhysicalConstants.h"
 #include "GaudiKernel/IRndmGenSvc.h" 
 
-//===========================================================================
-// Implementation file for class: MomentumRangeParticleGun
-//
-// 2008-05-18: Patrick Robbe adaptation to tool structure
-// 2014-06-22: Benedikt Hegner adaption for non LHCb use cases
-//===========================================================================
```

Comment:
If you want to modify it to be doxygen-compliant, fine. You can just list the authors and put the changes behind them? E.g.:
```c++
/**
  @author Patrick Robbe (adaptation to tool structure)
  @author Benedikt Hegner (adaption for non LHCb use cases)
  @date 2008-2016
*/
```